### PR TITLE
Redirect landing docs to hyperindex overview

### PR DESCRIPTION
**Pull request by moose-code**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -208,6 +208,10 @@ const redirectsList = [
     from: "/docs/HyperSync/hyperrpc-url-endpoints",
     to: "/docs/HyperSync/hyperrpc-supported-networks",
   },
+  {
+    from: '/',
+    to: '/docs/HyperIndex/overview',
```
Comment (DZakh) [inline docusaurus.config.js:213]: Maybe it should redirect to https://envio.dev ? I personally expected the behavior every time I clicked on the logo 😅

Comment (JasoonS) [reply docusaurus.config.js:213]: Yes, I agree with this :+1:

Comment (moose-code) [reply docusaurus.config.js:213]: Thanks! Agreed. 

So we are saying docs.envio.dev "/" redirects to envio.dev?

Comment (moose-code) [reply docusaurus.config.js:213]: Agree the logo should take us back to the site. 

But was thinking docs.envio.dev should just redirect to the hyperindex overview page?

Comment (JasoonS) [reply docusaurus.config.js:213]: Ah, we are actually talking about different things. I didn't look closely.

I think this redirect makes sense. But when clicking the 'envio' logo in the top left, it should go to the landing site.

Comment (JasoonS) [reply docusaurus.config.js:213]: This code isn't working though :grimacing: https://envio-docs-git-change-landing-docs-page-envio-9ec2d675.vercel.app/

No redirects happening.

Comment (moose-code) [reply docusaurus.config.js:213]: Yes I'm not sure why I haven't managed to get it working :( Fixing again now

**Comment by moose-code** [comment]
Clsoing this as. I've fixed here https://github.com/enviodev/docs/pull/401

**Pull request closed without merging**